core/src/main/java/com/linecorp/armeria/client/retry/RetryingClientBuilder.java

```diff
@@ -72,6 +73,21 @@ public T defaultMaxAttempts(int defaultMaxAttempts) {
     }
 
     /**
+     * Sets the {@code responseTimeoutMillisOfEachReq}. When requests in {@link RetryingClient} are made,
+     * corresponding responses are timed out by this value.
+     * It will be set by the default value in {@link Flags#defaultResponseTimeoutMillis()}, if the client
+     * dose not specify.
+     *
+     * @return {@link T} to support method chaining.
+     */
+    public T responseTimeoutMillisOfEachReq(long responseTimeoutMillisOfEachReq) {
```

Comment:
What about `responseTimeoutMillisForEachAttempt`?

Reply:
That's a good name. I'v been trying to figure out a better name, but I couldn't.